boards/imix/src/lowpan_frag_dummy.rs

```diff
@@ -399,41 +371,37 @@ impl<'a, A: time::Alarm + 'a> LowpanTest<'a, A> {
                                _: &[u8],
```

Comment:
I don't understand why this functions takes in this `_: &[u8]` argument which it does not appear to use at all. When `send_ipv6_packet()` is called, the `&MLP` reference is passed in here, but it still doesn't seem as though this function actually makes any use of it..

Reply:
I think this is an artifact from earlier revisions (in particular, I think that the function signature came from early revisions of the `sixlowpan_dummy.rs` file). The `MLP` buffer is for the mesh local prefix, and the test suite should probably be extended to test for different `MLP` values, which is why it was originally an argument to the function. But you're right, the function doesn't currently use it.